docs/error-reporting/platform-integrations/real-devices/setup.md

```diff
@@ -0,0 +1,128 @@
+- **Name**: error.type; **Format**: None; **Type**: String
+
+</TabItem>
+<TabItem value="ios" label="iOS">
+
+- **Name**: application.identifier; **Format**: None; **Type**: String
+- **Name**: device.machine; **Format**: None; **Type**: String
+- **Name**: error.type; **Format**: None; **Type**: String
+
+</TabItem>
+</Tabs>
+
+To add the attributes:
+1. In the Backtrace Console, go to **Project settings > Attributes**.
+1. Click **+**.
+1. Enter the values for the required fields.
+1. Click **Create**.
+1. Repeat steps 2 to 4 for each required attribute.
+
+For more information about attributes, see [Indexing Attributes](/error-reporting/project-setup/attributes/).
+
+## Integrate the Backtrace SDK
+<Tabs>
+<TabItem value="android" label="Android">
+
+To integrate the Backtrace SDK with your Android app, see [Setting Up Backtrace for Android](/error-reporting/platform-integrations/android/setup/).
+
+</TabItem>
+<TabItem value="ios" label="iOS">
+
+To integrate the Backtrace SDK with your iOS app, see the [iOS Integration Guide](https://support.backtrace.io/hc/en-us/articles/360040104692-iOS-Integration-Guide).
+
+</TabItem>
+</Tabs>
+
+## Link your Real Device Tests with Backtrace
+You’re now ready to link your real device tests in Sauce Labs with the crashes and errors reported to Backtrace. Once this is set up you will be able to directly view your crashes and errors in Sauce Labs.
+
+<Tabs>
+<TabItem value="admin" label="Sauce Labs Admin">
+
+This enables the Backtrace integration for your current data center.
+
+1. From your Sauce Labs account, go to the **Integrations** page (you need to have the role of organization admin to be able to navigate to this page).
+1. Click **Backtrace Enable**.
+1. Enter the **Project Name**, **Dashboard URL**, and **API Token**.
+1. Click **Save**.
+
+</TabItem>
+<TabItem value="user" label="Sauce Labs User">
+
+This enables the Backtrace integration for your entire organization. Each account will be able to make use of this feature.
+
+1. From your Sauce Labs account, go to any real device test result page (live or automated).
+1. From the bottom of the screen, select the **Error & Crash Reporting** tab.
+1. Click **Link with Backtrace**.
+1. Enter the **Project Name**, **Dashboard URL**, and **API Token**.
+1. Click **Save**.
+
+:::note
+Once enabled, you cannot update the integration settings or disable the integration. To do so, you need to contact your organization admin.
+:::
+</TabItem>
+</Tabs>
+
+:::note
+Currently, one subdomain, project, and API token is supported per organization.
+:::
+
+## Run your Real Device Test
+For information on how to run your real device test (live or automated), see [Mobile App Testing](/mobile-apps/).
+
+## View Test Results in Sauce Labs
+After running a test with the Backtrace SDK integrated into your app, you can view crashes and errors for your real device tests in Sauce Labs.
+
+1. Go to any real device test result page (live or automated).
```

Comment:
This step assumes that the test was run with an app that has Backtrace SDK integrated. Can we make it a bit more specific here? It's not any real device test result page but a specific one